src/widgets/benefits/ui/benefits.tsx

```diff
@@ -0,0 +1,61 @@
+import { memo } from 'react';
+import classNames from 'classnames/bind';
+import { Image } from '@/shared/ui/image';
+import { WidgetTitle } from '@/shared/ui/widget-title';
+import { Benefit, ImageLink, benefitMentorshipHome } from 'data';
+
+import styles from './benefits.module.scss';
+
+const cx = classNames.bind(styles);
+
+export type BenefitsProps = {
+  header?: string;
+  benefits?: Benefit[];
+  flex?: boolean;
+};
+
+type BenefitItemProps = {
+  classNames: string;
+  icon?: ImageLink;
+  text: string;
+};
+
+const shortBenefitMaxChars = 60;
+const BenefitItem = memo(({ text, icon, classNames }: BenefitItemProps) => (
+  <li className={classNames} data-testid="benefit">
+    {(icon?.href && icon?.alt) && <Image className={cx('benefit-icon')} src={icon.href} alt={icon.alt} />}
+    {text}
+  </li>
+));
+
+BenefitItem.displayName = 'BenefitItem';
```

Comment:
Do we really need `memo` and `displayName` here?

Reply:
Gave in to the rabbit's offer.)
It doesn't get you anywhere.

Done